### ISSUE TITLE
Copying not complete (103.4GB) to dst but removed from tmp drive due to dst full.

### ISSUE THREAD

**Issue by maxchock** [ericaltendorf/plotman#767]
I had a plot yesterday night, copied only 103.4GB because dst is full. But it still removed from tmp drive. The log file show these:

> Final File size: 101.375 GiB
Total time = 60031.384 seconds. CPU (86.000%) Thu Jun 10 02:27:04 2021
Copied final file from "/files/chiatemp/plot-k32-2021-06-09-09-46-40d2294988ce851a37432cd4d07b148eb3a8816a11d8750d7579031e3f9f4dd7.plot.2.tmp" to "/mnt/seagate5tb03/plot-k32-2021-06-09-09-46-40d2294988ce851a37432cd4d07b148eb3a8816a11d8750d7579031e3f9f4dd7.plot.2.tmp"
Copy time = 3074.608 seconds. CPU (19.480%) Thu Jun 10 03:18:19 2021
Removed temp2 file "/files/chiatemp/plot-k32-2021-06-09-09-46-40d2294988ce851a37432cd4d07b148eb3a8816a11d8750d7579031e3f9f4dd7.plot.2.tmp"? 1
Could not rename "/files/chiatemp/plot-k32-2021-06-09-09-46-40d2294988ce851a37432cd4d07b148eb3a8816a11d8750d7579031e3f9f4dd7.plot.2.tmp" to "/mnt/seagate5tb03/plot-k32-2021-06-09-09-46-40d2294988ce851a37432cd4d07b148eb3a8816a11d8750d7579031e3f9f4dd7.plot". Error No space left on device. Retrying in five minutes.

The copying process didn't return an error to plotman so it assume the copying was success and removed the temp plot?

Thanks & regards,
Max.

**Comment by altendky** [comment]
This activity is handled by the chia plot process, not plotman.  It often handles the lack of space gracefully and recovers, but not always.  Plotman doesn't clean up tmp files unless you ask it to.

**Comment by altendky** [comment]
I don't believe this is a plotman issue, rather a chia plotter issue as mentioned.  @maxchock, are you comfortable with that or should we discuss it more here?
